tests/base.py

```diff
@@ -43,13 +43,14 @@ def setUpClass(cls):
         print("\n\nTEST SETUP\n")
         cls.client = TestClient()
 
-    @classmethod
-    def tearDownClass(cls):
-        print("\n\nTEST TEARDOWN\n\n")
+    # @classmethod
+    # def tearDownClass(cls):
+    #     print("\n\nTEST TEARDOWN\n\n")
```

Comment:
Can you delete this code instead of leaving it commented out

Reply:
Done